fundamentals/middleware/problem-details-service/problem-details-service.csproj

```
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <RootNamespace>problem_detail_service</RootNamespace>
```

_nit:_ `PascalCase`? Or maybe remove it?

```suggestion
    <RootNamespace>ProblemDetailsServiceSample</RootNamespace>
```